apps/system/test/unit/secure_window_manager_test.js

```diff
@@ -88,6 +88,21 @@ suite('system/SecureWindowManager', function() {
         'the app was not registered in the maanger');
     });
 
+    test('App focused after opening', function() {
+      var stubFocus = this.sinon.stub(appFake, 'focus');
+      try {
+        window.secureWindowManager.handleEvent({type: 'secure-appcreated',
+                                                detail: appFake});
+        window.secureWindowManager.handleEvent({type: 'secure-appopened',
+                                                detail: appFake});
+        assert.isTrue(stubFocus.calledOnce,
+                      'the app was not given keyboard focus after opening');
```

Comment:
nits: The indentation looks a bit weird. Could you just make it align with the two spaces? I know this is very personal but we should keep the file with a unified style.